backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2Native.kt

```diff
@@ -25,7 +25,6 @@ import org.jetbrains.kotlin.cli.common.CLICompiler
 import org.jetbrains.kotlin.cli.common.CLITool
 import org.jetbrains.kotlin.cli.common.ExitCode
 import org.jetbrains.kotlin.cli.common.messages.CompilerMessageSeverity.ERROR
```

Comment:
Backtrace is not needed because it rethrows an exception. It will be processed then by CLICompiler.
The aim of this reporting to print as much info about current compilation and compiler  instead of just exception

One of our failures now looks like this
```
error: compilation failed: null cannot be cast to non-null type org.jetbrains.kotlin.ir.symbols.IrValueSymbol

 * Source files: helpers.kt, testUtils.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt, 1.kt, 2.kt, _launcher.kt
 * Compiler version info: Konan: EAP 0.6.0.0 / Kotlin: 1.2.0
 * Output kind: PROGRAM

exception: kotlin.TypeCastException: null cannot be cast to non-null type org.jetbrains.kotlin.ir.symbols.IrValueSymbol
        at org.jetbrains.kotlin.ir.util.IrUnboundSymbolReplacer.visitGetValue(IrUnboundSymbolReplacer.kt:414)
        at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitGetValue(IrElementTransformerVoid.kt:137)
        at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitGetValue(IrElementTransformerVoid.kt:24)
        at org.jetbrains.kotlin.ir.expressions.impl.IrGetValueImpl.accept(IrGetValueImpl.kt:41)
```